packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -158,7 +158,8 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
-    const { can } = useAbility()
+    const ability = useAbility()
+    const { can } = ability
```

Comment:
At least in the gh app I can't see that it gets actually used

Reply:
It's used, just not in the changed lines of code.